stylesheet.css

```diff
@@ -0,0 +1,32 @@
+/* Menu */
```

Comment:
On the second thought I agree that providing default stylesheet isn't the best option. My main idea is to make component as self-contained as it possible. I think there are some ways to make it with `purescript-css` but it's not blocking this pr definitely :)